ocp_resources/node_network_configuration_policy.py

```diff
@@ -440,6 +440,11 @@ def nnces(self):
                 nnces.append(nnce)
         return nnces
 
+    def node_nnce(self, node_name):
+        for nnce in self.nnces:
```

Comment:
return [nnce for nnce in self.nnces if  nnce.labels["nmstate.io/node"] == node_name]